src/Microsoft.DocAsCode.App/RunPdf.cs

```diff
@@ -13,9 +13,23 @@ namespace Microsoft.DocAsCode
 {
     internal static class RunPdf
     {
+        private static string GetFullFilePath(string filePath, string baseDirectory)
+        {
+            if (string.IsNullOrEmpty(filePath) || Path.IsPathRooted(filePath))
+            {
+                return filePath;
+            }
+
+            return Path.Combine(baseDirectory, filePath);
+        }
+
         public static void Exec(PdfJsonConfig config)
         {
-            var wkhtmltopdfFilePath = Path.Combine(config.BaseDirectory, config.Wkhtmltopdf?.FilePath);
+            EnvironmentContext.SetBaseDirectory(Path.GetFullPath(string.IsNullOrEmpty(config.BaseDirectory) ? Directory.GetCurrentDirectory() : config.BaseDirectory));
+            // TODO: remove BaseDirectory from Config, it may cause potential issue when abused
+            var baseDirectory = EnvironmentContext.BaseDirectory;
+
+            var wkhtmltopdfFilePath = GetFullFilePath(config.Wkhtmltopdf?.FilePath, baseDirectory);
```

Comment:
The `IsPathRooted` check is in the helper method should have the same effect as `Path.Combine` so I was removing the helper method, turns out causing `null` handling problems. Consider:

```suggestion
            var wkhtmltopdfFilePath = config.Wkhtmltopdf?.FilePath is null ? null : Path.Combine(config.Wkhtmltopdf.FilePath, baseDirectory);
```

Reply:
Sure, I'll remove the helper as well.